packages/components/src/Dialog/Layout/DialogLayout.story.tsx

```diff
@@ -74,16 +68,51 @@ Full.args = {
   ...Header.args,
 }
 
-const NoPaddingTemplate: Story<DialogLayoutExampleProps> = () => (
+const extraButton = (
+  <ButtonTransparent color="neutral" size="medium">
+    Cancel
+  </ButtonTransparent>
+)
+
+export const HeaderDetail = () => (
+  <Box bg="ui1">
+    <DialogLayout header="Header" headerDetail={extraButton} footer="Footer">
+      <ConstitutionShort />
+    </DialogLayout>
+  </Box>
+)
+
+export const HeaderCloseButton = () => (
+  <Box bg="ui1">
+    <DialogLayout header="Header" headerCloseButton={true} footer="Footer">
+      <ConstitutionShort />
+    </DialogLayout>
+  </Box>
+)
+
+export const FooterSecondary = () => (
+  <Box bg="ui1">
+    <DialogLayout
+      header="Header"
+      footer={
+        <DialogFooter secondary={extraButton}>
+          <Button>Primary</Button>
+        </DialogFooter>
+      }
```

Comment:
`DialogLayout` already wraps content output in `footer` within another `DialogFooter` so it's not needed here. Should instead be:

```suggestion
      footer={<Button>Primary</Button>
      footerSecondary="Secondary text"
```

Reply:
oops yep, that totally make sense, my bad!